lms/static/sass/_developer.scss

```diff
@@ -51,6 +51,19 @@
   text-align: center;
 }
 
+// UI: visual dividers
```

Comment:
Can you add a note that this came from the Instructor Dash so we know where to track it down once we bring these styles out of the developer sass area?